x-pack/plugin/searchable-snapshots/src/main/java/org/elasticsearch/index/store/cache/SparseFileTracker.java

```diff
@@ -37,6 +37,8 @@
 
     private final long length;
 
+    private final long recoveredBytesFromPersistentCache;
```

Comment:
I'm not super happy about this naming, any ideas to improve this?

Reply:
Maybe `initialLength` with a comment?